packages/@uppy/utils/src/NetworkError.js

```diff
@@ -0,0 +1,13 @@
+class NetworkError extends Error {
+  constructor (error = null, xhr = null) {
+    super(error.message)
+
+    this.isNetworkError = true
+    this.originalRequest = xhr
```

Comment:
Do we need the `xhr` argument & setting `.originalRequest` here?
`xhr` is never passed, `.originalRequest` is never used & we assign `error.request = xhr` in most places anyway.

Reply:
`originalRequest` can be used by developers who get `NetworkError`, so I’d probably keep it. Thanks for noticing it’s never passed, I forgot!